client/src/main/com/sinch/sdk/SinchClient.java

```diff
@@ -129,6 +131,13 @@ private void handleDefaultVoiceSettings(
               : builder.voiceRegion;
       builder.setVoiceUrl(props.getProperty(String.format("voice-server-%s", region.value())));
     }
+
+    // application management server
+    if (null == configuration.getVoiceApplicationMngmtUrl()
```

Comment:
Verbose Java with shortening feels outlandish :D 
```suggestion
    if (null == configuration.getVoiceApplicationManagementUrl()
```

Reply:
Fixed